github/teams.go

```diff
@@ -476,3 +476,88 @@ func (s *TeamsService) RemoveTeamProject(ctx context.Context, teamID int64, proj
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// GroupObject reprents list of Idp Group.
+type GroupObject struct {
+	Groups []*Group `json:"groups,omitempty"`
+}
+
+// Group represents an Idp group status.
+type Group struct {
+	GroupID          *string `json:"group_id,omitempty"`
+	GroupName        *string `json:"group_name,omitempty"`
+	GroupDescription *string `json:"group_description,omitempty"`
+}
+
+// ListIDPGroupsInOrganization list IdP groups available in an organization.
+//
+// Github API docs: https://developer.github.com/v3/teams/team_sync/
+func (s *TeamsService) ListIDPGroupsInOrganization(ctx context.Context, org string, opt *ListOptions) (*GroupObject, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/team-sync/groups", org)
+	u, err := addOptions(u, opt)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// TODO: remove custom Accept header when this API fully launches.
+	req.Header.Set("Accept", mediaTypeTeamSync)
+
+	groups := new(GroupObject)
+	resp, err := s.client.Do(ctx, req, groups)
+
+	if err != nil {
+		return nil, resp, err
+	}
+	return groups, resp, nil
+}
+
+// ListIDPGroupsForTeam list IdP groups connected to a team on GitHub.
+//
+// Github API docs: https://developer.github.com/v3/teams/team_sync/
```

Comment:
s/Github/GitHub/

URL should be: https://developer.github.com/v3/teams/team_sync/#list-idp-groups-for-a-team